gap/attr.gi

```diff
@@ -476,6 +476,12 @@ function(digraph)
   return GABOW_SCC(OutNeighbours(digraph));
 end);
 
+InstallMethod(DigraphNrStronglyConnectedComponents, "for a digraph",
+[IsDigraph],
+function(digraph)
+  return Length(DigraphStronglyConnectedComponents(digraph).comps);
```

Comment:
This works fine, but when you have a one line function like this, instead of using three lines as you have, you can write it shorter (and I think it's what we tend to do in the digraphs package):
```
InstallMethod(DigraphNrStronglyConnectedComponents, "for a digraph",
[IsDigraph],
digraph -> Length(DigraphStronglyConnectedComponents(digraph).comps));
```